### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by akucintavalent**
I followed the requirements given in [To Do list: add & remove](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md), after having done  [To Do list: list structure](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md) and [To Do list: interactive list](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md). You can see [Sneak peak: To Do list](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md).

### CODE REVIEW

File: src/add-item.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import toDoList from './to-do-list.js';
+import displayList from './display-items.js';
+import Task from './task.js';
+import updateItems from './status-updates.js';
+
+const addElementToListHandler = () => {
+  const inputField = document.querySelector('.input input');
+  const returnSymbol = document.querySelector('.return-symbol');
```
Comment (SajjadAhmad14) [inline src/add-item.js:9]: Users are unable to add task items to the list.

File: src/edit-description.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import toDoList from './to-do-list.js';
+
+const editDescription = (list = toDoList.data) => {
```
Comment (SajjadAhmad14) [inline src/edit-description.js:3]: Users should be able to edit task items.

File: src/clear-all-completed.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import toDoList from './to-do-list.js';
+
+const clearAllCompleted = () => {
+  const clearAllButton = document.querySelector('.clear-completed-button');
+  clearAllButton.addEventListener('click', () => {
```
Comment (SajjadAhmad14) [inline src/clear-all-completed.js:5]: Users should be able to check off items from the list.

File: src/display-items.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import toDoList from './to-do-list.js';
+
```
Comment (SajjadAhmad14) [inline src/display-items.js:2]: Please review all the requirements again and make sure everything is working as expected.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Dan, 
 ### You have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
README file is professional. :heavy_check_mark:
GitHub flow is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/display-items.js (hunk 4 above)
Comment (akucintavalent) [reply src/display-items.js:2]: I am sorry, but everything is working as expected on my PC.

Comment (SajjadAhmad14) [reply src/display-items.js:2]: Please watch this video: https://www.loom.com/share/a61d8643e5834c1485e083b0e85bb8da

Comment (akucintavalent) [reply src/display-items.js:2]: Thanks, I think I might have found the problem

Comment (mikethreels) [reply src/display-items.js:2]: - [ ] great job addressing the problem with adding and checking off items 👍 however the edit and remove functions are not working initially still. before the event listener for the 3 dot Icon is working I need to reload the page first. in order to solve this issue, you might want to try running the `editDescription()` function after adding an item.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Dan, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @akucintavalent,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**